app/views/Settings/index.tsx

```diff
@@ -155,6 +180,14 @@ const SettingsScreen = ({ navigation }: SettingsScreenProps): JSX.Element => {
           />
         </View>
 
+        <View style={styles.section}>
+          <SettingsListItem
+            label={t('screen_titles.delete_location_history')}
+            onPress={deleteLocationHistory}
```

Comment:
what do you think about `handleOnPressDeleteLocationHistory`  as a function name here?

Reply:
💯 changed!